examples/24_solar_battery_grid/README.md

```diff
@@ -0,0 +1,87 @@
+# Example 23: Solar Battery Grid System
+
+## Overview
+
+This example demonstrates a solar + battery + grid system that showcases the new unified grid component. The system can both buy electricity from the grid and sell excess electricity back to the grid using separate grid connection instances.
+
+## System Description
+
+### Technologies
+
+1. **Solar PV** (200 MWdc)
+   - Utility-scale solar with single-axis tracking
+   - 1.34 DC/AC ratio
+   - 65% bifacial panels
+   - Ground coverage ratio: 0.3
+
+2. **Battery Storage** (200 MWh, 25 MW charge/discharge rate)
+   - Simple generic storage model
+   - Demand-following control strategy (100 MW demand profile)
+   - Min SOC: 10%, Max SOC: 100%
+   - Charge/discharge efficiency: 100%
+
+3. **Grid Buy** (1000 MW interconnection)
+   - Buys electricity from grid to meet unmet demand
+   - Purchase price: $0.10/kWh
+   - No interconnection costs (set to $0)
+
+4. **Grid Sell** (1000 MW interconnection)
+   - Sells excess electricity back to grid
+   - Sell price: $0.00/kWh (no revenue in this configuration)
+   - No interconnection costs (set to $0)
+
+## Key Features Demonstrated
+
+### Unified Grid Component
+
+The grid component in this example is designed to handle both electricity purchases and sales within a single component. This allows for flexible configurations depending on the system's needs.
+
+**Key Features:**
+- Single component supports both buying and selling electricity
```

Comment:
I think this should go without saying (as in my above comment)

Reply:
Understood, hopefully the note above provides clarification